diff --git a/index.js b/index.js
index 67fa826d06306dbf5ef944187f2dcfcbed9a6714..f3390ca3689f79d65be4d3fd2a9b15ffec01e5b9 100644
--- a/index.js
+++ b/index.js
@@ -1,6 +1,6 @@
 'use strict'
 
-const stringWidth = require('string-width')
+const stringWidth = await import('string-width')
 
 function ansiAlign (text, opts) {
   if (!text) return text
@@ -50,7 +50,7 @@ ansiAlign.right = function right (text) {
   return ansiAlign(text, { align: 'right' })
 }
 
-module.exports = ansiAlign
+export default ansiAlign
 
 function halfDiff (maxWidth, curWidth) {
   return Math.floor((maxWidth - curWidth) / 2)
diff --git a/package.json b/package.json
index 96a9471c2c4acfea9d4bae0a8cdd1f628647fac9..bea2d348ee23d4e63e6a9274338614ce5fbae37b 100644
--- a/package.json
+++ b/package.json
@@ -2,6 +2,7 @@
   "name": "ansi-align",
   "version": "3.0.1",
   "description": "align-text with ANSI support for CLIs",
+  "type": "module",
   "main": "index.js",
   "scripts": {
     "pretest": "standard",
